client/me/help/help-contact/index.jsx

```diff
@@ -694,6 +683,8 @@ export default connect(
 			ticketSupportConfigurationReady: isTicketSupportConfigurationReady( state ),
 			ticketSupportEligible: isTicketSupportEligible( state ),
 			ticketSupportRequestError: getTicketSupportRequestError( state ),
+			hasMoreThanOneSite: get( getCurrentUser( state ), 'visible_site_count', 0 ) > 0,
```

Comment:
There's also a [`getCurrentUserSiteCount`](https://github.com/Automattic/wp-calypso/blob/332aaf4d8e28a702ee9f3cd09db6b17aab279343/client/state/current-user/selectors.js#L55-L68) selector, but that uses `site_count` (instead of `visible_site_count`). Would it make sense to use that here? Which of either corresponds better to the previous `sites.get().length > 1` check?